purchase_request_to_rfq/wizard/purchase_request_line_make_purchase_order.py

```diff
@@ -0,0 +1,278 @@
+            'price_unit': 0.0,
+            'product_qty': qty,
+            'account_analytic_id': item.line_id.analytic_account_id.id,
+            'purchase_request_lines': [(4, item.line_id.id)],
+        }
+        if item.line_id.procurement_id:
+            vals['procurement_ids'] = [(4, item.line_id.procurement_id.id)]
+        self._execute_purchase_line_onchange(vals)
+        return vals
+
+    @api.model
+    def _get_order_line_search_domain(self, order, item):
+        vals = self._prepare_purchase_order_line(order, item)
+        order_line_data = [('order_id', '=', order.id),
+                           ('product_id', '=', item.product_id.id or False),
+                           ('product_uom', '=', vals['product_uom']),
+                           ('account_analytic_id', '=',
+                            item.line_id.analytic_account_id.id or False),
+                           ]
+        if not item.product_id:
+            order_line_data.append(('name', '=', item.name))
+        if not item.line_id.procurement_id and \
+                item.line_id.procurement_id.location_id:
+            order_line_data.append(
+                ('location_id', '=',
+                 item.line_id.procurement_id.location_id.id))
+        return order_line_data
+
+    @api.multi
+    def make_purchase_order(self):
+        res = []
+        purchase_obj = self.env['purchase.order']
+        po_line_obj = self.env['purchase.order.line']
+        pr_line_obj = self.env['purchase.request.line']
+        purchase = False
+
+        for item in self.item_ids:
+            line = item.line_id
+            if item.product_qty <= 0.0:
+                raise exceptions.Warning(
+                    _('Enter a positive quantity.'))
+
+            location = line.request_id.picking_type_id.default_location_dest_id
+            if self.purchase_order_id:
+                purchase = self.purchase_order_id
+            if not purchase:
+                po_data = self._prepare_purchase_order(
+                    line.request_id.picking_type_id, location,
+                    line.company_id)
+                purchase = purchase_obj.create(po_data)
+
+            # Look for any other PO line in the selected PO with same
+            # product and UoM to sum quantities instead of creating a new
+            # po line
+            domain = self._get_order_line_search_domain(purchase, item)
+            available_po_lines = po_line_obj.search(domain)
+            if available_po_lines:
+                po_line = available_po_lines[0]
+                new_qty = pr_line_obj._calc_new_qty(line, po_line=po_line)
+                if new_qty > po_line.product_qty:
+                    po_line.product_qty = new_qty
+                    # Leave the purchase order order to calculate its prices
+                    # itself
+                    po_line._onchange_quantity()
+                    po_line.purchase_request_lines = [(4, line.id)]
```

Comment:
This line should be executed regardless if new_qty > po_line.product_qty. \

When you create two PR lines for 1 unit for imac, which has a minimum qty of 5, now the PO line is only linked to PR 1, not to PR2. That is incorrect.

This issue also occurs in 9.0 and 8.0